test/integration/node-specific/crypto_callbacks.test.ts

```diff
@@ -25,24 +25,35 @@ const SKIP_AWS_TESTS = [AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_REGION, AW
 describe('cryptoCallbacks', function () {
   let client: MongoClient;
   let sandbox;
-  before(function () {
-    sandbox = sinon.createSandbox();
-  });
+
+  const hookNames = new Set([
+    'aes256CbcEncryptHook',
+    'aes256CbcDecryptHook',
+    'randomHook',
+    'hmacSha512Hook',
+    'hmacSha256Hook',
+    'sha256Hook'
+  ]);
 
   beforeEach(function () {
     if (SKIP_AWS_TESTS) {
       this.currentTest?.skip();
       return;
     }
 
-    sandbox.restore();
+    sandbox = sinon.createSandbox();
+
```

Comment:
Is it possible to stub the whole object in one go? 

`sandbox.spy(cryptoCallbacks)`